src/components/EntryHeader/EntryHeader.js

```diff
@@ -1,10 +1,20 @@
+import { classNames as cn } from 'utils';
 import { FeaturedImage, Heading, PostInfo } from 'components';
 
 import styles from './EntryHeader.module.scss';
-
+/**
+ * A Page or Post entry header component
+ * @param {Props} props The props object.
+ * @param {string} props.title The post/page title.
+ * @param {MediaItem} props.image The image node.
+ * @param {string} props.date The post/page publish date.
+ * @param {string} props.author The post/page author's name.
+ * @param {string} props.className An optional className to be added to the EntryHeader.
+ * @return {React.ReactElement} The EntryHeader component.
+ */
 export default function EntryHeader({ title, image, date, author, className }) {
   const hasText = title || date || author;
-  const entryHeaderClasses = [styles['entry-header'], className].join(' ');
+  const entryHeaderClasses = cn([styles['entry-header'], className]);
```

Comment:
Unrelated to the scope of this PR but I just caught that we are assigning `cn()` to a variable in this component but using `cn()` directly in `className` in other components. Thoughts on aligning to one or the other? @theodesp @apmatthews @blakewilson

Reply:
Both are valid, but in cases where there are more that two classes assigned using a variable is preferable. For example in Button.js:
```
let classNames = cn([
    styles.button,
    styles[`button-${buttonStyle}`],
    className,
  ]);
```

Reply:
Consistency is nice, but in this case I don't mind using `cn()` slightly differently depending on the context (as Theo mentioned).

@theodesp Could we split the class name updates out to a separate PR though? I love the changes, but it would be preferable to keep this PR limited to the scope of MERL-216.

Reply:
Rebased without this commit.